ePBFT/src/pbft/libbyz/Replica.cpp

```diff
@@ -229,13 +229,101 @@ void Replica::receive_message(const uint8_t* data, uint32_t size)
   }
 }
 
+bool Replica::compare_execution_results(
```

Comment:
compare_replay_execution_results

Reply:
This is used in execute_tentative also so it should be more generic